conda/core/subdir_data.py

```diff
@@ -442,7 +443,7 @@ def _process_raw_repodata(self, repodata):
                 (conda_packages.items(), True),
                 (((k, legacy_packages[k]) for k in use_these_legacy_keys), False)):
             for fn, info in group:
-                duplicate_info = deepcopy(info)
+                duplicate_info = deepcopy(info) if signature_verification.enabled else None
```

Comment:
If you add this as a workaround then it might be good to add a comment detailing the "why" this is in place.